packages/components/src/border-control/types.ts

```diff
@@ -161,12 +160,11 @@ export type DropdownProps = ColorProps & {
 	 */
 	showDropdownHeader?: boolean;
 	/**
-	 * Start opting into the larger default height that will become the
-	 * default size in a future version.
+	 * Size of the control.
 	 *
-	 * @default false
+	 * @default default
```

Comment:
```suggestion
	 * @default 'default'
```

I believe the README also needs to be updated